### PR TITLE
Introduces fill attendance functionality

### PR DESCRIPTION
**Pull request by ghatighorias**

### CODE REVIEW

File: web/controllers/attendance_controller.ex [hunk 1]
```diff
@@ -40,8 +52,58 @@ defmodule CoursePlanner.AttendanceController do
...
+    when role in ["Coordinator", "Teacher"] do
+
+      attendances_data =
+        classes
+        |> Map.values()
+        |> Enum.map(fn(class_value) ->
+             Map.values(class_value["attendances"])
+           end)
+        |> List.flatten
+
```
Comment (rhnonose) [inline web/controllers/attendance_controller.ex]: You can merge these two pipelines as one :)

File: web/controllers/attendance_controller.ex [hunk 2]
```diff
@@ -40,8 +52,58 @@ defmodule CoursePlanner.AttendanceController do
...
+  end
+
+  def update_fill(%{assigns: %{current_user: %{role: role}}} = conn,
+       %{"offered_course" => %{"classes" => classes}, "attendance_id" => offered_course_id})
+    when role in ["Coordinator", "Teacher"] do
+
+      attendances_data =
+        classes
+        |> Map.values()
+        |> Enum.map(fn(class_value) ->
```
Comment (ggpasqualino) [inline web/controllers/attendance_controller.ex]: you may use `Enum.flat_map` instead of `Enum.map(...) |> List.flatten`

Comment (ghatighorias) [inline web/controllers/attendance_controller.ex]: Done

File: web/controllers/attendance_controller.ex (hunk 1 above)
Comment (ghatighorias) [inline web/controllers/attendance_controller.ex]: Done

Comment (ggpasqualino) [inline web/controllers/attendance_controller.ex]: is it? 🤔

**Pull request merged**